_data/navigation.yml

```diff
@@ -37,6 +37,10 @@
       new_window: false
       external: false
       img_src: /assets/images/nav-icons/Mail.svg
+    - name: Recognition
+      link: https://badges.layer5.io/
+      new_window: true
```

Comment:
Please add external: true

Reply:
Actually alignment doesn’t looks good, I can do it

Reply:
@Raj100679 do it for now

Reply:
@iArchitSharma will do it in some time

Reply:
Looks good to me
![image](https://github.com/meshery/meshery.io/assets/74408634/0658232a-33e6-4bea-9179-2028a711d345)